.github/workflows/docker-image.yml

```diff
@@ -18,9 +18,19 @@ jobs:
     - name: Get current tag
       id: vars
       run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
+    - name: Extract Major and Minor Versions
+      id: version
+      run: |
+        FULL_VERSION=${{ steps.vars.outputs.tag }}
+        MAJOR_VERSION=$(echo $FULL_VERSION | cut -d '.' -f 1)
+        MINOR_VERSION=$(echo $FULL_VERSION | cut -d '.' -f 1,2)
+        echo ::set-output name=major::${MAJOR_VERSION}
+        echo ::set-output name=minor::${MINOR_VERSION}
```

Comment:
Could you elaborate on what these do?

Reply:
As I understood, it writes values into github "variables"

Reply:
not trivial to test without actually running CI, though

Reply:
yes these lines set the 'major' and 'minor' output parameters, similar to how the 'tag' parameter is set in the 'Get current tag' step. These outputs are then used in subsequent steps of the workflow.